base/scene.py

```diff
@@ -22,5 +23,5 @@ def render(self):
     def handle_input(self):
         pass
 
-    def transition_to(self, scene_name):
-        self.scene_manager.transition_to(scene_name)
+    def transition_to(self, scene_name, **kwargs):
```

Comment:
Some scenes will require additional data and we can't guess what in advance.
The kwargs allow us a great flexibility.
How should I proceed?